### ISSUE TITLE
Option for Overriding own schedule

### ISSUE THREAD

**Issue by Leololololo** [calcom/cal.com#5779]
I run a Computer Support Business. Most of my Customers are booking via my digital calendar integration on my website.
Unfortunately, many customers also call me for an appointment because they are not able to book via website, or they are just too lazy.

My Schedule is usually fully booked, and sometimes I want to squeeze some of these calling customers in my schedule in the next few days. Unfortunately, my own online bookings prevent me to add their mostly quick appointment in my schedule via the booking site, and I'm only able to add them manually via my calendar service of choice. This obviously cuts them out from the features of automatic appointment reminders etc.

I think it would be an easy solution to just add an "override schedule" button (just for me when I'm logged in) in the booking page, so I can overbook a schedule. I know for example that acuity scheduling of Squarespace does that with a simple checkmark on the booking page for logged in admin users.

<img width="894" alt="Example" src="https://user-images.githubusercontent.com/63395206/204613163-7f348f69-6353-4c67-a916-d29319b5a19c.png">

**Comment by FormlosLK** [comment]
Maybe another caption for this checkmark could be "**ignore all my schedules and all my appointments in any connected calender**" to make it even more clear. (To be honest, of course, my proposal is much too long. But I just wanted to make it as clear as possible.)

**Comment by PeerRich** [comment]
quite a niche feature. I'll keep this as low priority until more people mention something similar

**Comment by RainerZufall9393** [comment]
Would also be very fond of this feature. Used to use it regularly with Acuity. @PeerRich

**Comment by PeerRich** [comment]
can you not use the date overwrite feature in https://app.cal.com/availability?

**Comment by Leololololo** [comment]
> can you not use the date overwrite feature in https://app.cal.com/availability?

Here's an example of the problem: my schedule is fully booked tomorrow from 8:00 to 17:00. 

However, a customer who is over 60 years old calls me and requests a quick appointment to fix an issue that I didn't completely resolve the day before. I would like to accommodate the customer by giving them a 10-minute appointment, but I cannot do so because my schedule is already overbooked.

Ideally, I would like to be able to freely assign this customer a quick appointment, even though the time slot is already full. 
The person who originally booked the full slot knows anyways that an appointment can start 10 minutes later because i have a flexible clause .

**Comment by RainerZufall9393** [comment]
This is a very good example! @Leololololo 

With me it is so that I always have my fixed appointment times, but sometimes a short-term opportunity arises and then I would like to be able to plan the appointment completely flexible.

To give you an example of me. My regular times are 12:00-20:00 and you have to book with me the appointments at least 1 day in advance.

However, as a host, I would like to have the option, like Acuity, to book an appointment for today, in the morning at 09:00. Even if I already have an appointment manually or via Cal.com, I would like to be able to overbook it. Also regardless of any buffer times and regardless of which of my calendar groups (Private, Business, etc.) the appointment is in.

Complete freedom for us as hosts to book an appointment at any time, regardless of all the restrictions we have set for the participants normally.

As I said, this is a great feature that I miss a lot since I'm no longer with Acuity and I don't think anyone else offers this yet either.

**Comment by PeerRich** [comment]
i am referring to this button: 

![CleanShot 2023-03-21 at 10 28 30@2x](https://user-images.githubusercontent.com/8019099/226565449-8a3a7e7d-bfa5-4c79-985f-c0ebd68a45cc.jpg)

does this not do what you want?

**Comment by Jaibles** [comment]
> i am referring to this button:
> 
> ![CleanShot 2023-03-21 at 10 28 30@2x](https://user-images.githubusercontent.com/8019099/226565449-8a3a7e7d-bfa5-4c79-985f-c0ebd68a45cc.jpg)
> 
> does this not do what you want?

This doesn't allow adding an appointment, which is what the op is requesting as far as I understand. The date overrides will only allow you to block yourself. They actually want the ability to unblock the schedule to add in one appointment.

**Comment by cpunnelliehealth** [comment]
+1 for this request. Sometimes our users just want to be able to schedule a session regardless of their availability and any conflicts.

**Comment by vquelque** [comment]
+1 for this request. Especially, I would really like the ability to reschedule appointments outside working hours. It happens that I have to move meetings around and that I want to reschedule a meeting with someone outside of my working hours to accommodate everything. 
Adding this and the ability to check calendar conflicts on a per-event type basis would make cal.com the best appointment-scheduling app.

**Comment by eigenstil** [comment]
seems like the same idea or case like in #11842

**Comment by EduNetEurope** [comment]
> quite a niche feature. I'll keep this as low priority until more people mention something similar

Two more requests:
https://github.com/calcom/cal.com/issues/14608
https://github.com/calcom/cal.com/issues/11842

An updates?

**Comment by guidol-ive** [comment]
Thanks for the tip on Acuity!

Our scenario is very similar: the synced calendar of a professional is filled with internal and external meetings, and time reservations for things to do. When a customer calls with an urgent issue, the professional should be able to start the workflow himself on top of an internal meeting or planned thing to do (including video integration and billing) with cal.com. Otherwise everything needs to be done manually, easily taking 30 minutes of overhead for a call of one hour.

Possible workaround: each time tweak the agenda to make space available in the calendar, but takes time to synchronize, while the customer is on the phone waiting.

Possible solution: add "ignore calendar availability" option during planning when the user using the schedule window is identical to the owner of the schedule.

Largely or partially related issues:

- https://github.com/calcom/cal.com/issues/8744
- https://github.com/calcom/cal.com/issues/12244
- https://github.com/calcom/cal.com/issues/21467

**Comment by matheins** [comment]
We need this as well

**Comment by jannesblobel** [comment]
1+ any updates? 
and  Bypass availability check fro team mates as well
